### PR TITLE
Register new package Dolo.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [EconForge/Dolo.jl](https://github.com/EconForge/Dolo.jl)
Release: [v0.1.0](https://github.com/EconForge/Dolo.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/EconForge/Dolo.jl/builds/244608294)
cc: @sglyon

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@sglyon This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

File: Dolo/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+DataStructures
+DataFrames
+AxisArrays
+NLsolve
+Optim
+Requests
+QuantEcon 0.10.1
+Distributions
+ForwardDiff
+Compat 0.25.0
```
Comment (tkelman) [inline Dolo/versions/0.1.0/requires]: doesn't look like you're using anything newer than `abstract type` which is 0.17

Comment (sglyon) [inline Dolo/versions/0.1.0/requires]: Good catch. I'll fix that when I land in Berkeley in a couple hours

**Comment by sglyon** [comment]
OK, I think this is now ready. Thanks!

File: Dolo/versions/0.1.0/requires [hunk 2]
```diff
@@ -0,0 +1,12 @@
+julia 0.5
+MacroTools
+YAML 0.2.1
+DataStructures
+AxisArrays
+NLsolve
+Optim
+Requests
+QuantEcon 0.10.1
```
Comment (tkelman) [inline Dolo/versions/0.1.0/requires]: this absolutely requires the newest versions of both YAML and QuantEcon, and wouldn't work with anything earlier even on julia 0.5?

Comment (sglyon) [inline Dolo/versions/0.1.0/requires]: Yep, that's right. 

We need custom tag support in YAML and Mvn sampler type in quantecon

**Comment by sglyon** [comment]
Tests pass locally but fail on Travis. I'm going to try to debug and will post here when I figure it out.

**Pull request merged**